Fix bug in News index page on Chrome

mybits opened the pull request:
Chrome renders text in html element differently from FF and Safari. That was causing a problem
with changing a dimension of the news tile height, what was finally the reason of gaps in the next tiles row as the they were pushed down.
Setting fixed height on a tile container should be a solution.

(+ minor fixes in css intendation)

Screenshot of a bug:
<img width="1045" alt="screen shot 2016-03-03 at 22 03 29" src="https://cloud.githubusercontent.com/assets/5237879/13511136/d3362a7c-e18b-11e5-96f6-bce9534e4eaa.png">

mybits committed:
Chrome renders text in html element differently from FF and Safari. That was causing a problem
with changing a dimension of the news tile height, what was finally the reason of gaps in the next tiles row as
the they were pushed down.
Setting fixed height on a tile container should be a solution.

(+ minor fixes in css intendation)

tomsabin commented on source/assets/stylesheets/styles/_news_tile.scss:
```diff
@@ -3,15 +3,19 @@
+  @extend .blog-medium-tile__container;
+
+  @include breakpoint($tablet-breakpoint) {
+    height: 386px;
```
I suggest changing the `.blog-medium-tile__container` min-height to `388px` instead of having this override as the problem may occur in blogs too but more so to prevent overlapping margins which this fixed height introduces:

with `height` (notice the 2px margin overflow)
<img width="389" alt="screen shot 2016-03-04 at 12 34 39" src="https://cloud.githubusercontent.com/assets/885223/13527560/b8c3871c-e207-11e5-87ee-4463df6a0555.png">

with `min-height` adjusted
<img width="446" alt="screen shot 2016-03-04 at 12 35 49" src="https://cloud.githubusercontent.com/assets/885223/13527561/b8c40962-e207-11e5-9cc2-895a8510c91c.png">

mybits committed:
Increase min-heigth rather to set it to a fixed value in news and blog tiles container
(that was to fix a bug in Chrome which is processing the text in a way the tile dimension might change)

mybits commented:
thanks @tomsabin for your comment!

tomsabin commented:
LGTM
